### DISCUSSION TITLE
블로그 개선 (1) - 글 파일 구조 및 이미지 개선

### DISCUSSION THREAD

**Discussion by giscus** [yiyb0603/yiyb-blog#69] [Blog Comment]
# 블로그 개선 (1) - 글 파일 구조 및 이미지 개선

최근 제 블로그의 글 파일 구조 및 이미지를 개선한 경험을 공유합니다.

https://yiyb-blog.vercel.app/posts/improve-posts-directory-and-images

**Comment by lee-sihun** [comment]
잘 읽고 갑니다!
